{"name": "test_get_user_by_id[Get User - Empty User ID]", "status": "failed", "statusDetails": {"message": "AssertionError: FAILURE: \u001b[31massert 200 == 404\ntests\\test_api.py:104 in test_get_user_by_id() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1", "trace": "FAILURE: \u001b[31massert 200 == 404\ntests\\test_api.py:104 in test_get_user_by_id() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1"}, "attachments": [{"name": "log", "source": "2dab5a6e-c1bc-4fec-827b-a44623b32997-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'Case': 'Get User - Empty User ID', 'user_id': '', 'is_valid': False}"}], "start": 1670168748506, "stop": 1670168749292, "uuid": "422eb634-416e-4b79-9676-3ff7ff305048", "historyId": "f1c7c3f666b45b74f0f6194e811d4985", "testCaseId": "0348809fddc28b0397e61969c6de4b71", "fullName": "tests.test_api.TestApi#test_get_user_by_id", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "DESKTOP-EGG762M"}, {"name": "thread", "value": "11972-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}